policyengine_us/variables/gov/states/ri/tax/income/agi/subtractions/taxable_retirement_income/ri_retirement_income_subtraction.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class ri_retirement_income_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island Taxable retirement income subtraction"
```

Comment:
remove `taxable` from labels and parameter folders

subtractions are of taxable elements by definition